### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dcrankshaw**
First draft of the new README. I'd welcome any feedback or comments. Feel free to either make comments in the PR or to directly edit the branch and push to this branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,96 @@
...
-[![License](http://img.shields.io/:license-Apache%202-red.svg)](LICENSE)
+## Project Overview
 
-## About the Project
+![Clipper System Overview](images/clipper_arch.png)
 
-![Clipper System Overview](images/arch_diagram.png)
+## What is Clipper?
 
+Clipper is a prediction serving system that sits between user-facing applications and a wide range of commonly used machine learning frameworks.
```
Comment (atumanov) [inline README.md]: is it frameworks, or models would be better?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,96 @@
...
 
-Machine learning is being deployed in a growing number of applications which demand real-time, accurate, and robust predictions under heavy query load. However, most machine learning frameworks and systems only address model training and not deployment.
+## What does Clipper do?
 
-Clipper is the first general-purpose low-latency prediction serving system.  Interposing between end-user applications and a wide range of machine learning frameworks, Clipper introduces a modular architecture to simplify model deployment across frameworks.  Furthermore, by introducing caching, batching, and adaptive model selection techniques, Clipper reduces prediction latency and improves prediction throughput, accuracy, and robustness without modifying the underlying machine learning frameworks.
+* Clipper **simplifies integration of machine learning techniques** into user facing applications by providing a simple standard prediction and feedback REST interface across a wide range of commonly used machine learning frameworks.  *Clipper makes product teams happy.*
```
Comment (atumanov) [inline README.md]: by providing a simple, standard REST interface for prediction and feedback across...

File: README.md [hunk 3]
```diff
@@ -1,38 +1,96 @@
...
-### Additional Resources
+* Clipper **simplifies model deployment** and **helps reduce common bugs** by using the same tools and libraries used in model development to render live predictions.  *Clipper makes data scientists happy.*
 
-+ [Design Doc (work-in-progress)](https://docs.google.com/document/d/1Ghc-CAKXzzRshSa6FlonFa5ttmtHRAqFwMg7vhuJakw/edit?usp=sharing)
-+ [Research Paper](https://arxiv.org/abs/1612.03079)
 
-Clipper is a project in the UC Berkeley [RISE Lab](https://rise.cs.berkeley.edu/).
 
-![RISE Lab logo](images/rise_lab_logo.png)
+* Clipper **improves throughput** and ensures **reliable millisecond latencies** by introducing adaptive batching, caching, and straggler mitigation techniques.  *Clipper makes the infra-team less unhappy.*
```
Comment (atumanov) [inline README.md:20]: Clipper reliably supports latencies on the order of tens of milliseconds by ...

File: README.md [hunk 4]
```diff
@@ -1,38 +1,96 @@
...
+
+* Clipper **improves prediction accuracy** by introducing state-of-the-art bandit and ensemble methods to intelligently select and combine predictions and achieve real-time personalization across machine learning frameworks.  *Clipper makes users happy.*
```
Comment (atumanov) [inline README.md:22]: Clipper maximizes prediction accuracy by

File: README.md [hunk 5]
```diff
@@ -1,38 +1,96 @@
...
+Deploying trained machine-learning models into production today is an ad-hoc, labor-intensive, and error-prone process. This creates an enormous impediment to building and maintaining user-facing applications that incorporate machine-learning.
+
+Clipper is designed to simplify this process by decoupling applications that
+consume predictions from trained models that produce predictions.
+Clipper is a robust, high-performance serving system that can scale to thousands of requests per second and provide 
+responses that meet latency service level objectives on the order of milliseconds.
+As a result, Clipper can be safely incorporated into a production serving stack without negatively
+impacting application latencies.
+
+Simultaneously, Clipper allows data scientists to easily deploy trained models to production.
```
Comment (atumanov) [inline README.md]: At the same time, Clipper allows...

File: README.md [hunk 6]
```diff
@@ -1,38 +1,96 @@
...
+Data science is an iterative process, and simplifying the model deployment process allows
+data scientists to more easily experiment with new features and models to quickly improve
+application accuracy. Data scientists deploy models to Clipper with the same code used for
+training, eliminating a common class of bugs in machine-learning that arise from code duplication.
+And Clipper supports deploying models trained in many machine learning frameworks and implemented
+in many programming languages to support the rich ecosystem of data science tools available today.
```
Comment (atumanov) [inline README.md]: in a variety of programming languages to support

File: README.md [hunk 7]
```diff
@@ -41,63 +99,77 @@ $ git clone --recursive https://github.com/ucbrise/clipper.git
...
+__Building Clipper:__
 
-First generate the cmake files with `./configure`. This generates an out-of-source build directory called `debug`.
-Go into one of this directory and then run `make` to actually
-compile the code. You should only need to re-configure if you change one of the `CMakeLists.txt` files.
+First generate the CMake files with `./configure`. This generates an out-of-source build directory called `debug`.
+Go into this directory and then run `make` to actually
+compile the code. You should only need to re-run the configuration script if you change one of the `CMakeLists.txt` files.
 To build for release, run `./configure --release` which generates the `release` build directory instead of debug.
 If you want to clean everything up, you can run `./configure --cleanup`.
```
Comment (atumanov) [inline README.md]: `./configure --cleanup-quiet` suppresses dialog while cleaning up.

**Review by atumanov — APPROVED** [review]
I have a few wording suggestions attached to this review. Otherwise, LGTM.

**Comment by jegonzal** [comment]
What is the status of this PR?  It would be good to get this merged and create the GitHub.io page by Spark Summit talk.  Are we ready to merge?

**Pull request merged**